package.json

```diff
@@ -85,5 +86,8 @@
     "typescript": "^5.0.3",
     "typescript-docs-verifier": "^2.4.0",
     "webpack": "^5.77.0"
+  },
+  "resolutions": {
+    "dns-over-http-resolver": "^2.1.1"
```

Comment:
Need to add custom resolution for now until the dependency tree is updated

`warning "workspace-aggregator-7fe22546-d232-4ae9-9884-df7a13280865 > @lodestar/beacon-node > @multiformats/multiaddr > dns-over-http-resolver > native-fetch@4.0.2" has unmet peer dependency "undici@*".`

This PR fixes the warnings and is included in 2.1.1
- https://github.com/vasco-santos/dns-over-http-resolver/pull/78